lib/packetfilter.py

```diff
@@ -240,8 +240,10 @@ def _GenerateAddrStatement(self, addrs, exclude_addrs):
   def _GeneratePortStatement(self, ports):
     port_list = []
     for port_tuple in ports:
-      for port in port_tuple:
-        port_list.append(str(port))
+      if port_tuple[0]==port_tuple[1]:
+        port_list.append(str(port_tuple[0]))
+      else:
+        port_list.append(str(port_tuple[0])+':'+str(port_tuple[1]))
```

Comment:
Nit: looking at the rest of the source, it seems the coding style uses spaces around binary operators. For consistency, would you mind adapting the patch format accordingly?